packages/sources/css/src/components/card/src/index.css

```diff
@@ -105,6 +101,7 @@
 
 .vtmn-card_variant--full-image img {
   height: 100%;
+  min-width: rem(200px);
```

Comment:
Why?

Reply:
To avoid having the content going out of the image, but maybe I can refacto this

Reply:
Indeed, we need to discuss about this. Tomorrow ? :)